index.html

```diff
@@ -80,7 +83,12 @@
  			<div id="loader"></div>
  			<div id="loader"></div>
  	</div>
-
+	
+	<div class="loader-wrapper" id="loader-9">
+		  	<div id="loader">
+		 		<center><img src="http://i.imgur.com/rwZMEGa.png"></center>
```

Comment:
Don't you think it would be better to upload the image to the repository itself and put that as src?